src/yz_entropy_mgr.erl

```diff
@@ -502,6 +503,7 @@ tick(S) ->
     S3 = lists:foldl(fun(_,SAcc) ->
                              maybe_poke_tree(SAcc)
                      end, S2, lists:seq(1,10)),
+    update_throttle(S3),
```

Comment:
You can't tell just by looking at the diff here, but that call to `update_throttle` is actually part of the `tick` function that is called periodically (I think it's once every second). So basically we update the throttle delay time based on the current level of load, and this happens on every timer tick.